resources/testharness.js

```diff
@@ -16,14 +16,12 @@ policies and contribution forms [3].
 (function (global_scope)
 {
     var debug = false;
-    // default timeout is 10 seconds, test can override if needed
```

Comment:
I think you want to keep this - it's about the file-level timeout.

Reply:
Ok, I've kept a comment and reworded it to:
```js
    // Default timeout is 10 seconds, test can extend it to 60 seconds if needed.
```